### DISCUSSION TITLE
How to add Modal button after modal content

### DISCUSSION THREAD

**Discussion by dheerajKsharma** [breezefront/community#172] [1. Help] [answered]
Hi,
How can I add modal button after modal content section. So it will visible in footer of modal. Currently it shows in header of modal which is of no use.
https://breezefront.com/docs/modal#modal
![image](https://github.com/breezefront/community/assets/132253509/9c063503-098e-4000-a3f1-087180c98b73)

**Comment by vovayatsyuk** [comment]
I don't know why it's happening. Maybe it's your custom styles?

```
$($0).modal({
    autoOpen: true,
    buttons: [{
        text: $t('Ok'),
        class: '',
        attr: {},
    }, {
        text: $t('Cancel'),
        class: '',
        attr: {},
    }]
});
```

Result:

<img src="https://github.com/breezefront/community/assets/306080/ab7c2532-38fa-4f57-ad66-f192945a55ef" width="589"/>

**Reply by dheerajKsharma** [reply]
Actually it's due to popup type. When I use popup as type it works well. But when I use 'slide' as type it shows button inside header. 
Any fix?

**Reply by vovayatsyuk** [reply, marked as answer]
Looks like a bug in Magento template [slide template](https://github.com/magento/magento2/blob/HEAD/app/code/Magento/Ui/view/base/web/templates/modal/modal-slide.html). Here is result of slide type on Luma:

<img src="https://github.com/breezefront/community/assets/306080/5ba714f0-ff6a-408c-a44a-db81ca561817" width="300"/>

As a workaround, I recommend emulating slide view using HTML attributes:

```js
// create and show modal
$(el).modal({
    autoOpen: true,
    buttons: [{
        text: $t('Ok'),
        class: '',
        attr: {},
    }, {
        text: $t('Cancel'),
        class: '',
        attr: {},
    }]
});

// transform popup into slide
$(el).modal('instance').modal
    .removeClass('modal-popup')
    .addClass('modal-slide')
    .attr('data-type', 'slide')
```

**Reply by dheerajKsharma** [reply]
Woahh!!! Thanks.
Worked like a charm :)
